pkg/cfg/cfg.go

```diff
@@ -62,18 +65,23 @@ func (ia IDPAccount) String() string {
   SessionDuration: %d
   Profile: %s
   RoleARN: %s
-}`, appID, ia.URL, ia.Username, ia.Provider, ia.MFA, ia.SkipVerify, ia.AmazonWebservicesURN, ia.SessionDuration, ia.Profile, ia.RoleARN)
+}`, appID, policyID, ia.URL, ia.Username, ia.Provider, ia.MFA, ia.SkipVerify, ia.AmazonWebservicesURN, ia.SessionDuration, ia.Profile, ia.RoleARN)
 }
 
 // Validate validate the required / expected fields are set
 func (ia *IDPAccount) Validate() error {
-	if ia.Provider == "OneLogin" {
+	switch ia.Provider {
+	case "OneLogin":
```

Comment:
Is there a reason your updating the onelogin related code?

Reply:
Ideally you would keep your change just to the scope of adding your provider, and then any other fixes would be seperate.

Reply:
I missed that I changed the message in the onelogin block.  I'll remove this and resubmit.  Thanks.